doc/consultant_notes.rst

```diff
@@ -22,3 +22,34 @@ only use them like described here.
 For youtube only, if you want to put a link to a youtube video like 
 'https://www.youtube.com/watch?v=somevid' rewrite it to 
 'https://www.youtube.com/embed/somevid' instead
+
+Integrating a motion.ai chatbot
+========================================
+
+Generate the bot css by running
+::
+  botBackgroundColor=whatever ./scripts/gen_chatbot_css.sh
+  
+The available variables are:
+::
+  botBackgroundColor
+  botTextColor
+  botTimeColor
+  humanBackgroundColor
+  humanTextColor
+  humanTimeColor
+  
+Valid values are css colors. If the output is not printed to the terminal, it should be copied to your clipboard.
+
```

Comment:
Instead of spaces like this, you can use bullets in rst, since your language is very sparse.

```
* Point 1
* Point 2
* Point 3```